osf_tests/metrics_factories.py

```diff
@@ -0,0 +1,50 @@
+import factory
+from elasticsearch_metrics.factory import MetricFactory
+from osf.metrics import PreprintDownload, PreprintView
+from osf_tests.factories import PreprintFactory
+
+
+class PreprintDownloadFactory(MetricFactory):
```

Comment:
Woops, these are no longer used, since the tests don't actually send data to ES any more. Remove this file.